airflow/api/common/experimental/trigger_dag.py

```diff
@@ -77,20 +77,20 @@ def _trigger_dag(
     if conf:
         run_conf = conf if isinstance(conf, dict) else json.loads(conf)
 
-    triggers = []
-    dags_to_trigger = [dag] + dag.subdags
-    for _dag in dags_to_trigger:
-        trigger = _dag.create_dagrun(
+    dag_runs = []
+    dags_to_run = [dag] + dag.subdags
```

Comment:
I'm wondering if this is even correct. Shouldn't subdags be handled by the parent DAG's SubDagOperator?

Reply:
I agree. That should be looked at seperately. This is just a rename so it's easier to see what's happening

Reply:
I can take a look to that issue seperately if it's ok?

Reply:
Shall I mark this as resolved and discuss it in another PR? I'll investigate this subdags execution later